shared-module/audiofilters/Filter.c

```diff
@@ -209,6 +233,11 @@ audioio_get_buffer_result_t audiofilters_filter_get_buffer(audiofilters_filter_o
         channel = 0;
     }
 
+    // Update filter_states if filters list has been appended or removed
+    if (self->filters->len != self->filter_states_len) {
+        reset_filter_states(self);
+    }
```

Comment:
once it's a tuple, this is not needed right?

Reply:
Yep. This will be omitted in the next commit.